.stylish-haskell.yaml

```diff
@@ -0,0 +1,63 @@
+steps:
+  - simple_align:
+      cases: true
+      top_level_patterns: true
+      records: true
+
+  # Import cleanup
+  - imports:
+      align: global
+      list_align: after_alias
+      pad_module_names: true
+      long_list_align: inline
+      empty_list_align: inherit
+      list_padding: 4
+      separate_lists: true
+      space_surround: false
+
+  # Language pragmas
+  - language_pragmas:
+      style: vertical
+      align: true
+      remove_redundant: true
+      language_prefix: language
+
+  # Remove trailing whitespace
+  - trailing_whitespace: {}
+
+columns: 100
+newline: native
+cabal: true
+language_extensions:
```

Comment:
Why is there a list of language extensions here?

Reply:
This is for the parser, AFAIK